### DISCUSSION TITLE
What is the rectangular shaped PCB antenna for ?

### DISCUSSION THREAD

**Discussion by jaytxrx** [OpenI6X/opentx#319] [Q&A] [answered]
Hi,

I noticed that there is a rectangular kind of antenna marking on the PCB (please see the attached image - bottom right part)
Is this really an antenna or just a marking to solder some module ? Looks like some Wifi or Bluetooth antenna marking.
Does anyone have any clue what is it for ?

![image](https://user-images.githubusercontent.com/431506/206931826-3e973532-e9e1-4764-8b51-0c9bc4a99cf4.png)

**Comment by ajjjjjjjj** [comment, marked as answer]
Unpopulated place most probably designed for BT module.
